packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DataVisAnnotationTask/DataVisAnnotationTask.js

```diff
@@ -55,12 +54,11 @@ class DataVisAnnotationTask extends React.Component {
       annotations,
       task
     } = this.props
-    let annotation
-    if (annotations && annotations.size > 0) {
-      annotation = annotations.get(task.taskKey)
-    }
+    // let annotation
+    // if (annotations && annotations.size > 0) {
+    //   annotation = annotations.get(task.taskKey)
+    // }
```

Comment:
Can this just be deleted?

Reply:
The TODO comment suggests it's being worked on, so I'm not sure.

Reply:
They can be removed for now, but let's make sure there is an open issue about finishing it.

Reply:
I've added #999 to track this